### PR TITLE
US10_bar_menu

### PR DESCRIPTION
**Pull request by Smetankaa**
// Here trying to comment

**please review, if there any q let me know**

### CODE REVIEW

**Comment by batounen** [comment]
can you please provide more detailed info on all these 201 bugs?

File: src/US10_bar_menu [hunk 1]
```diff
@@ -0,0 +1,5 @@
+//US10_bar_menu
+
+//Bar menu is tested
+
+//201 bugs found
```
Comment (batounen) [inline src/US10_bar_menu:5]: can you please provide more detailed info on all these 201 bugs? ty

File: src/US10_bar_menu [hunk 2]
```diff
@@ -0,0 +1,5 @@
+//US10_bar_menu
```
Comment (batounen) [inline src/US10_bar_menu:1]: new code comment

File: src/US10_bar_menu (hunk 1 above)
Comment (Smetankaa) [reply src/US10_bar_menu:5]: yes, this and that bla bla US

File: src/US10_bar_menu (hunk 2 above)
Comment (batounen) [reply src/US10_bar_menu:1]: can you please add one line of code to complete your TC

Comment (batounen) [reply src/US10_bar_menu:1]: ok good!

File: src/US10_bar_menu [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+//Bar menu is tested
+
+//201 bugs found
```
Comment (AlenaKrets) [inline src/US10_bar_menu:5]: comment

Comment (AlenaKrets) [inline src/US10_bar_menu:5]: my code here

**Review by AlenaKrets — CHANGES_REQUESTED** [review]

**Review by AlenaKrets — APPROVED** [review]

**Review by batounen — APPROVED** [review]

**Pull request merged**